js_modules/dagit/packages/core/src/assets/LaunchAssetChoosePartitionsDialog.tsx

```diff
@@ -435,6 +435,12 @@ const LaunchAssetChoosePartitionsDialogBody: React.FC<Props> = ({
               }}
               padding={{vertical: 12, horizontal: 24}}
             >
+              {'anchorAssetKey' in target && (
```

Comment:
😬

Reply:
Hmm maybe I'll change this to a `target.type === 'pureWithAnchorAsset' for consistency? Not sure what you want to do here!